test-action.txt

```diff
@@ -0,0 +1 @@
+Abcd
```

Comment:
Not sure what the story is here, but I don't think this file shouldn't have been committed.

Reply:
Let's pretend no one ever saw that... Sorry about that, I was triggering master commits to test the workflow and it ended up leaking to the branch when I fetched + merged the changes!